app/Http/Controllers/RoomController.php

```diff
@@ -19,6 +19,13 @@ public function index(Request $request)
         $bot = Bot::findOrFail($request->bot_id);
         $chatwork = Chatwork::withAPIToken($bot->bot_key);
         $rooms = $chatwork->rooms()->list();
-        return $rooms;
+        $group_boxs = [];
+        foreach($rooms as $room) {
+            if ($room['type'] == "group") {
+                array_push($group_boxs, $room);
+            }
+        }
+
+        return $group_boxs;
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Http/Controllers/RoomController.php</br>**Line**: 29</br>Variable "group_boxs" is not in valid camel caps format